spec/Section 6 -- Execution.md

```diff
@@ -526,31 +813,51 @@ CollectFields(objectType, selectionSet, variableValues, visitedFragments):
     - If {DoesFragmentTypeApply(objectType, fragmentType)} is {false}, continue
       with the next {selection} in {selectionSet}.
     - Let {fragmentSelectionSet} be the top-level selection set of {fragment}.
-    - Let {fragmentGroupedFieldSet} be the result of calling
-      {CollectFields(objectType, fragmentSelectionSet, variableValues,
-      visitedFragments)}.
+    - If {deferDirective} is defined:
+      - Let {path} be the corresponding entry on {deferDirective}.
```

Comment:
should this be `label` instead of `path`?

Reply:
a950a96